srcpkgs/perl-Test-Differences/template

```diff
@@ -1,15 +1,16 @@
-# Template build file for 'perl-Test-Differences'.
+# Template file for 'perl-Test-Differences'
 pkgname=perl-Test-Differences
 version=0.64
-revision=1
+revision=2
+noarch=yes
 wrksrc="${pkgname#*-}-${version}"
 build_style=perl-module
 hostmakedepends="perl perl-Text-Diff"
 depends="${hostmakedepends}"
+checkdepends="${hostmakedepends} perl-Capture-Tiny"
 short_desc='Test::Differences - Test strings / data and show differences if not ok'
 maintainer="Enguerrand de Rochefort <voidlinux@rochefort.de>"
-homepage="https://metacpan.org/release/Test-Differences"
 license="Artistic, GPL-1, GPL-2, GPL-3"
```

Comment:
license="Artistic-1.0-Perl, GPL-1.0-or-later"

Please see if the GPL-2 and GPL-3 apply and if they are -or-later or -only